lib/streamlit/elements/arrow_table.py

```diff
@@ -0,0 +1,325 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018-2019 Streamlit Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pandas as pd
+import pyarrow as pa
+from streamlit import util
+from streamlit import type_util
+
+
+def marshall(proto, data, default_uuid):
```

Comment:
I would have the output param as the last one, but I could not find any authoritative doc on this...

Reply:
And what's the intuition behind that?